### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by weyert**
Added a callout to the credentials to the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ The objective of this repository is hold the most common configuration for `verd
 
 > Feel free to contribute whether you considere any use case is relevant for the public in general.
 
+## Login 
+
+If you want to login into the Verdaccio instance creating via these Docker Examples, please try:
```
Comment (sergiohgz) [inline README.md]: creating - > creat**ed**

**Review by sergiohgz — CHANGES_REQUESTED** [review]
Only that, the rest :+1:

**Comment by weyert** [comment]
Updated :)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ The objective of this repository is hold the most common configuration for `verd
 
 > Feel free to contribute whether you considere any use case is relevant for the public in general.
 
+## Login 
+
+If you want to login into the Verdaccio instance created via these Docker Examples, please try:
+
+Username: jpicado
+Password: jpicado
+
```
Comment (sergiohgz) [inline README.md]: Sorry, I didn't show that this is the main README. This user-pass pair doesn't apply on all examples, As @juanpicado said in Discord, the README you should update is in _docker-local-storage-volume_ folder ([here](https://github.com/verdaccio/docker-examples/blob/master/docker-local-storage-volume/readme.md))

Also, there is other example that need this info, _https-portal-example_  ([here](https://github.com/verdaccio/docker-examples/blob/master/https-portal-example/README.md)). Please, can you add in this README also? Thanks :slightly_smiling_face:

**Review by sergiohgz — CHANGES_REQUESTED** [review]
Minor change. Also, squash commits please.

**Comment by weyert** [comment]
I have updated the text. I didn't get it to squash commits.

**Review by sergiohgz — APPROVED** [review]
👌 Perfect, thanks

**Pull request merged**